src/Illuminate/Auth/SessionGuard.php

```diff
@@ -419,13 +430,17 @@ public function attemptWhen(array $credentials = [], $callbacks = null, $remembe
      */
     protected function hasValidCredentials($user, $credentials)
     {
-        $validated = ! is_null($user) && $this->provider->validateCredentials($user, $credentials);
+        return $this->timebox->make(function ($timebox) use ($user, $credentials) {
```

Comment:
Please add a check if the timebox is not `null` because the timebox can be null (see constructor)

Reply:
There should probably also be a test if timebox is null and the expected behaviour

Reply:
See above comment. Now the property cannot be null.